.travis.yml

```diff
@@ -23,7 +23,9 @@ install: true
 script:
   - export GO15VENDOREXPERIMENT=1
   - if [ $TRAVIS_OS_NAME == 'linux' ]; then
-      cd shared && npm i --global flow-bin@`tail -n1 .flowconfig` && flow &&
+      cd shared && npm i -g flow-bin@`tail -n1 .flowconfig` && flow &&
+      cd ../desktop && npm i -g babel-cli && npm run setup-ci-tools && npm run setup-dev-tools &&
+      cd ../shared && eslint . && 
```

Comment:
Yeah, it's running out of `shared` so we don't waste time scanning the other dirs.  eslint finds the eslintrc in the parent dir fine:
```sh
 λ pwd                          client/shared (cjb/DESKTOP-1017-eslint ⚡) caius
/home/cjb/go/src/github.com/keybase/client/shared

 λ strace -f eslint . 2>&1 | grep eslintrc | head -n1 
[pid  9307] stat("/home/cjb/go/src/github.com/keybase/client/.eslintrc", {st_mode=S_IFREG|0664, st_size=999, ...}) = 0
```